jovian/__init__.py

```diff
@@ -23,7 +23,40 @@
 
 
 def reset():
-    """Reset the tracked hyperparameters & metrics (for a fresh experiment)"""
+    """Reset the tracked hyperparameters & metrics (for a fresh experiment)
+    Example
+        .. code-block::
+
+            import jovian
+
+            data = {
+                'path': '/datasets/mnist',
+                'description': '28x28 images of handwritten digits (in grayscale)'
+            }
+            jovian.log_dataset(data)
+
+            hyperparams = {
+                'arch_name': 'cnn_1',
+                'lr': .001
+            }
+            jovian.log_hyperparams(hyperparams)
+
+            metrics = {
+                'epoch': 1,
+                'train_loss': .5,
+                'val_loss': .3,
+                'acc': .94
+            }
+            jovian.log_metrics(metrics)
+
+            jovian.reset()
+
+            newData = {
+                'path': '/datasets/mnist',
+                'description': 'a new set of data post-reset'
+            }
+            jovian.log_dataset(newData)
```

Comment:
Lets stick to just reset's usage in the Example section. As the rest of the things will be available in the same page of the docs. You can remove anything other than `jovian.reset`.

Here is the preview of that page : https://prajwalprashanth-demo.readthedocs.io/en/dw-doc-reset/jvn/logger.html